package.json

```diff
@@ -41,8 +41,9 @@
     "@typescript-eslint/parser": "^4.28.1",
     "@vuepress/plugin-google-analytics": "^1.8.2",
     "archiver": "^5.3.0",
-    "chart.js": "^3.4.0",
-    "chartjs-test-utils": "^0.3.0",
+    "chart.js": "^3.8.0",
+    "chartjs-chart-matrix": "^1.1.1",
```

Comment:
You changed the `package_lock.json` format, can you re-generate it using version 1 so I can understand exactly what dependencies have changed. I will update dependencies in a separate PR later and will switch to a more recent node/npm version.